### PR TITLE
Expands "Move" button to support moving multiple files together.

### LINKED ISSUE #942: Support for moving multiple selected items to the same destination.
It would be nice to be able to perform a bulk file move by selecting multiple files and Moving them to a new folder destination.

### PR DESCRIPTION
**Pull request by NHDaly**
"Move" button now displays for 1 or more selected items, and attempts to
move all of them to the entered destination. For each move that fails,
an error is displayed, but the other items are still moved.

Closes #942.

### CODE REVIEW

File: notebook/static/tree/js/notebooklist.js [hunk 1]
```diff
@@ -820,24 +816,31 @@ define([
...
+                                    body: $('<div/>')
+                                        .text("An error occurred while moving \"" + item_name + "\" from \"" + item_path + "\" to \"" + new_path + "\".")
+                                        .append($('<div/>')
+                                            .addClass('alert alert-danger')
+                                            .text(e.message || e)),
+                                    buttons: {
+                                        OK: {'class': 'btn-primary'}
+                                    }
+                                });
+                                console.warn('Error durring moving :', e);
```
Comment (minrk) [inline notebook/static/tree/js/notebooklist.js]: s/durring/during/

Comment (NHDaly) [inline notebook/static/tree/js/notebooklist.js]: Fixed. I also fixed this typo throughout the file (7 lines) in a595e43.

**Comment by minrk** [comment]
Thanks!

**Comment by NHDaly** [comment]
:D :dancer:

**Pull request merged**